### PR TITLE
Feature/progress bar

### PR DESCRIPTION
**Pull request by jsp-up**

### CODE REVIEW

**Review by Bellico — DISMISSED** [review]

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-progress-bar/kal-progress-bar.component.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  /**
+   * Value of the progress-bar. Defaults to zero. Max to 100
+   */
+  @Input()
+  get value(): number {
+    return this.progressValue;
+  }
+  set value(newValue: number) {
+    this.progressValue = clamp(coerceNumberProperty(newValue));
+  }
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-progress-bar/kal-progress-bar.component.ts:56]: trigger detection changes on new value

File: src/app/atoms/progress-bar/progress-bar.component.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+<label for="progressBarValue">Progress bar value :</label>
+<input id="progressBarValue"
+       type="number"
+       min="0"
+       max="100"
+       (keyup)="setValue($event)">
+
+<label for="progressBarColor">Progress bar color :</label>
+<input id="progressBarColor"
+       type="text"
```
Comment (ydomenjoud) [inline src/app/atoms/progress-bar/progress-bar.component.html]: type="color"

**Review by ydomenjoud — CHANGES_REQUESTED** [review]
fix comments

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-progress-bar/kal-progress-bar.component.ts (hunk 1 above)
Comment (jsp-up) [reply projects/kalidea/kaligraphi/src/lib/atoms/kal-progress-bar/kal-progress-bar.component.ts:56]: Already did that in onChanges

**Review by ydomenjoud — APPROVED** [review]

**Pull request merged**